static/templates/message_controls.hbs

```diff
@@ -9,6 +9,12 @@
     </div>
     {{/unless}}
 
+    {{#unless should_display_quote_and_reply}}
```

Comment:
You are still not passing `should_display_quote_and_reply` to this template and it will always be `undefined`,

Reply:
Could you Please elaborate, how `should_display_quote_and_reply` would be undefined, Thanks.

Reply:
See `should_display_quote_and_reply` is included in the args which is passed to `render_actions_popover_content` (for `actions_popover_content.hbs` file) [here](https://github.com/zulip/zulip/blob/da8817889bbd024656b39db0c98f30b95bd4de82/static/js/popovers.js#L554). Similarly you have to pass it in the `message_controls` template as well, otherwise the template does not know its value and considers it as undefined.